Mage/src/main/java/mage/constants/ComparisonType.java

```diff
@@ -5,8 +5,10 @@
  */
 public enum ComparisonType {
     FEWER_THAN("<", "fewer", "than"),
+    FEWER_THAN_OR_EQUAL_TO("<=", "fewer", "than or equal to"),
```

Comment:
I've thought about doing this for a while since there are _so many cards_ that have to use a literal value one off from the value in the printed text due to the >= and <= operators not being in this enum.

Maybe it's time to do so! The main downside I see is autocompletions are no longer unique from the first character.

The text1, text2 fields appear to only be used in a single condition, and that text is only relevant to a single card. Probably those fields should be removed entirely and conditions can generate the text as needed from the enum according to their particular needs.

Also, these names are a bit long, I might go with `OR_LESS`; `OR_GREATER` to steer clear of the existing names and match how they are typically used in card text. (Admittedly not the case for the text on this particular card.)